### PR TITLE
Add an API for querying metrics

### LINKED ISSUE #5: Add an API for asking about metrics
Client applications (e.g. Purl) should be able to supply a druid and get the relevant metrics as JSON.

### PR DESCRIPTION
**Pull request by thatbudakguy**
<s>Depends on #3 </s>
Closes #5

### CODE REVIEW

**Comment by peetucket** [comment]
Looks like some of the changes in this PR are also in #4 .... maybe merge that first and rebase if needed?

File: spec/controller/metrics_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe MetricsController, type: :controller do
```
Comment (jcoyne) [inline spec/controller/metrics_controller_spec.rb:5]: Can you make this a requests spec instead?  I believe controller specs are deprecated.

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/controller/metrics_controller_spec.rb (hunk 1 above)
Comment (thatbudakguy) [reply spec/controller/metrics_controller_spec.rb:5]: Will do, thx

Comment (thatbudakguy) [reply spec/controller/metrics_controller_spec.rb]: Done

**Review by jcoyne — APPROVED** [review]

**Pull request merged**